Library/Formula/twoping.rb

```diff
@@ -0,0 +1,18 @@
+class Twoping < Formula
+  desc "Ping utility to determine directional packet loss"
+  homepage "http://www.finnie.org/software/2ping/"
+  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
+  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
+
+  depends_on "Python"
```

Comment:
Looks like this works with Python 2.6 and 2.7, so you should be able to rely on the version of Python that comes with the OS.